test/runtime/ferguson/cache-remote-mock-tests/mock-ra-perf.skipif

```diff
@@ -1 +1,7 @@
-CHPL_RT_LOCALES_PER_NODE>1
+#!/usr/bin/env python3
+
+import os
+
+lpn = os.getenv("CHPL_RT_LOCALES_PER_NODE", default="1")
+
+print(int(lpn)>1)
```

Comment:
So I don't know who would even do this, but `CHPL_RT_LOCALES_PER_NODE` could be a non-int value which would raise a `ValueError`. Just to make sure this script never errors out like that:

```suggestion
print(int(lpn) > 1 if lpn.isdigit() else False)
```